est/001.est

```diff
@@ -0,0 +1,22 @@
+date: 28-02-2016
+author: Dali Freire
+method: champions.pert
+scope:
+  1: Infrastructure setup
+  2: Design the operation commands
+  3: GitHub/Issue trackers integration
+  4: Project site
+  5: Integrate with continuous integration systems
```

Comment:
What do you mean by that exactly, Rultor runs its own builds. There are ways to integrate with CI systems like discussed [here](http://www.yegor256.com/2015/03/29/rultor-with-appveyor.html) for Windows based builds.
But from my perspective this is not in the scope of the Rultor project, but rather the users choice in Dockerimage/script  isn't it?
But maybe I'm misunderstanding something here :), so what specifically is to be coded/was already coded for this point.